webextension/background.js

```diff
@@ -86,7 +86,14 @@ class StudyLifeCycleHandler {
         break;
       }
 
-      browser.cloudstorage.init(browser.runtime.getURL("./skin/clouddownloads.css"), interval).then(
+      browser.cloudstorage.onRecordTelemetry.addListener(
+        (value) => {
+          console.log("onRecordTelemetry", value);
```

Comment:
We really shouldn't ship with `console.log` still in the code...

Reply:
+1, removed console here. Couple that are still there are during initialization in background.js, will be updating those with shield utils integration.